drivers/node/node.go

```diff
@@ -27,6 +27,15 @@ const (
 	Directory FindType = "d"
 )
 
+// StoragePool is the storage pool structire on the node
+type StoragePool struct {
+	UUID                string
+	TotalSize           uint64
```

Comment:
embed the existing StoragePool structure returned by SDK API and add other fields that test needs.

This will be useful when we need to start looking at other fields in the pools (like labels) in future. We want to avoid redeclaring fields that are already present in api.StoragePool.